### ISSUE TITLE
Ошибка трансформации в формат EDT

### ISSUE THREAD

**Issue by kuzja086** [firstBitMarksistskaya/jenkins-lib#83]
Агент поднят на Windows, получаю следующую ошибку:
![Снимок](https://user-images.githubusercontent.com/51016001/170095777-f20b0e24-5c32-42d7-af7e-cf109dda1ae4.JPG)
Помогает исправление, но не знаю на сколько это правильно.
![image](https://user-images.githubusercontent.com/51016001/170095552-9a105a06-1463-46c0-87a1-4e67284efd22.png)

**Comment by nixel2007** [comment]
Подскажите, агент с едт у вас не на той же машине, что master/built-in, да?

**Comment by kuzja086** [comment]
> Подскажите, агент с едт у вас не на той же машине, что master/built-in, да?

Нет, master на другой машине под Линукс. Под Виндой Агент с метками "agent" и "edt"

**Comment by nixel2007** [comment]
Ага, понял. Действительно, через File так обращаться нельзя, просмотрел.

**Comment by kuzja086** [comment]
> Ага, понял. Действительно, через File так обращаться нельзя, просмотрел.

Такая же проблема с генерацией отчетов Allure, когда агент с меткой "agent" поднят на Windows. НЕ формирует путь к данным для формирования отчета

**Comment by atai** [comment]
Тоже самое с [ru/pulsar/jenkins/library/steps/EdtToDesignerFormatTransformation.groovy:38 ](https://github.com/firstBitMarksistskaya/jenkins-lib/blob/e219077e9e2a8c88be63499656d5bb9e24ec889b/src/ru/pulsar/jenkins/library/steps/EdtToDesignerFormatTransformation.groovy#L38) - getCanonicalPath() добавляет "/" перед буквой диска. Мастер и агент на разных ОС.
